HomeWorkSolution/Task3/Program.cs

```diff
@@ -8,29 +8,64 @@ namespace Task3
 {
     class Program
     {
-        const double bankRate = 0.0016;
+        const double bankRate = 0.02;
         static void Main(string[] args)
         {
-            decimal startDeposit = 5000;
+            decimal startDeposit = 1000;
             DateTime depositeDate = new DateTime(2019, 9, 1);
-            CalculateDeposit(startDeposit, depositeDate, 120);
+            bool isHighPrecision = new bool();
+            Console.WriteLine("Enter monthes count");
+            uint monthesCount = ReadAnswerFromUser();
+            Console.WriteLine("isHighPrecision false:");
+            CalculateDeposit(startDeposit, depositeDate, monthesCount, isHighPrecision);
 
+            isHighPrecision = true;
+            Console.WriteLine("\nisHighPrecision true:");
+            CalculateDeposit(startDeposit, depositeDate, monthesCount, isHighPrecision);
+            
             Console.ReadLine();
         }
 
         /// <summary>
         /// Calculate deposit.
         /// </summary>
-        /// <param name="startDeposit">Start value.</param>
+        /// <param name="currentDeposit">Start value.</param>
         /// <returns>Return deposit value included bank rate.</returns>
-        static void CalculateDeposit(decimal startDeposit, DateTime depositeDate, int monthCount)
+        static void CalculateDeposit(decimal currentDeposit, DateTime depositeDate, uint monthCount, bool isHighPrecision)
         {
+            Console.WriteLine(string.Format("{0} Deposit money at start: {1}; Bank rate: {2}% ", depositeDate.ToShortDateString(), currentDeposit, bankRate));
             decimal addedMoney;
```

Comment:
In a String.Format() method need to chenge to "Bank rate: {2:P2}" to see % better. Didn't see that before. 
Typing here cz don't want to do new commit.